.github/workflows/docs.yml

```diff
@@ -0,0 +1,32 @@
+name: "Docs"
+on:
+  push:
+    branches:
+      - 'develop'
```

Comment:
I wonder if this should actually be the LTS or the one behind develop. As develop things are new and very much edge edge.

Reply:
Fair point. At the moment it needs to be develop, as this is the only branch which has the `juju documentation` command. But long term, it makes sense to use the latest LTS branch. (Or even version the docs by branch, but that's another discussion).